src/CodeGen_LLVM.h

```diff
@@ -492,6 +492,11 @@ class CodeGen_LLVM : public IRVisitor {
     llvm::Constant *embed_constant_expr(Expr e);
 
     llvm::Function *add_argv_wrapper(const std::string &name);
+
+    llvm::Value *codegen_predicated_dense_vector_load(const Load *load, llvm::Value *vpred);
```

Comment:
I slightly prefer naming this codegen_dense_vector_load, and making the vpred argument optional ( = nullptr), since the function handles dense non-predicated loads as well.